Include/internal/pycore_global_objects.h

```diff
@@ -323,12 +354,276 @@ struct _Py_global_objects {
             _PyLong_DIGIT_INIT(255), \
             _PyLong_DIGIT_INIT(256), \
         }, \
+        \
+        .bytes_empty = _PyBytes_EMPTY_INIT, \
+        .bytes_characters = { \
```

Comment:
Why are we generating these tables by hand?

Reply:
I wanted to deal with code generation separately.